official/vision/beta/projects/yolo/optimization/sgd_torch.py

```diff
@@ -0,0 +1,339 @@
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import array_ops
+from tensorflow.keras.optimizers.schedules import LearningRateSchedule
+from tensorflow.python.training import gen_training_ops
+
+import tensorflow as tf
+import re
+import logging
+
+try:
```

Comment:
remove import, it is okay to have this implementation here.

Reply:
done